src/relayserver/RelayServer.ts

```diff
@@ -555,7 +569,8 @@ latestBlock timestamp   | ${latestBlock.timestamp}
       fromBlock: this.lastScannedBlock + 1,
       toBlock: 'latest'
     }
-    return await this.contractInteractor.getPastEventsForHub(topics, options)
+    const events = await this.contractInteractor.getPastEventsForHub(topics, options)
+    return events
```

Comment:
Idea complains about it as 'redundant', do you really need it?

Reply:
not sure - doesn't complain for me..